dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.optional.jaxb-2.3.feature

```diff
@@ -6,7 +6,7 @@ IBM-App-ForceRestart: uninstall, install
 IBM-Process-Types: client, server
 Subsystem-Name: Java XML Bindings 2.3
 IBM-API-Package: \
-  javax.activation; type="spec", \
+  javax.activation; require-java:="9"; type="spec", \
```

Comment:
I thought for 2.3 we required activation 1.1 independent of java 9 or not because jaxb 2.3 requires activation 1.1.

Reply:
Going to leave this for now to avoid different versions of the same Activation spec being available when running on JDK 8